pootle/pootle.conf

```diff
@@ -0,0 +1,201 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import os
+
+# Site title
+POOTLE_TITLE = os.environ.get('POOTLE_TITLE', 'My Pootle Translation Server')
+
+# Local time zone for this installation. Choices can be found here:
+# http://en.wikipedia.org/wiki/List_of_tz_zones_by_name
+# although not all choices may be available on all operating systems.
+# If running in a Windows environment this must be set to the same as your
+# system time zone.
+TIME_ZONE = 'Europe/Berlin'
+
+# Make this unique, and don't share it with anybody.
+SECRET_KEY = os.environ.get('SECRET_KEY', '4JY+9C9kEWASJk5MVkYH8ynd5ka9aRq56jssTwa2key8CZrSipNi1vD/1lPfzcqx/UY=')
```

Comment:
"Make this unique, and don't share it with anybody" :laughing:

Reply:
The key here is a fallback if the SECRET_KEY environment variable isn't set on the machine, which it should be.